### DISCUSSION TITLE
"Add Item" feature in project command has a regression

### DISCUSSION THREAD

**Discussion by viviankc** [community/community#53511] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

Issue:

Creating an issue through the feature "add item" in a project does not fill in the title anymore.

Repro:
1. Go to a github project
2. Click in a column "Add Item"
3. Type "# name_of_repository" and press "Enter"
4. Start typing the name of the issue and click "Create new issue # title of issue"
![image](https://user-images.githubusercontent.com/17960375/233813759-6af6e52e-3091-4a30-b6ff-dc92df5dc715.png)
5. The title does not fill in the issue anymore:
![image](https://user-images.githubusercontent.com/17960375/233813769-3c8f58b7-b760-4062-945a-ead084f950d4.png)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by IpkMg01** [comment]
This is happening to me too +1
